### ISSUE TITLE
feat: automate csv assignment inside enclave.json

### ISSUE THREAD

**Issue by qascade** [qascade/dcr#45]
## Description
CSVs can't be moved inside enclaves unless explicitly mentioned inside the enclave.json.  Check `files` field in `service/temp_enclave.json`. I want that to be automated and to be picked from Yaml, not through temporary CSV's. 
The enclave.json is to modified after ego-sign using sources specified in yaml. 

## How are you planning to resolve this issue?
1. Use basic json Writers to manipulate the file. 
2. Use some templates and use pongo2 templating engine.

**Comment by harshsingh32** [comment]
I would like to work on this issue, please assign this issue to me under GSSoC'23.

**Comment by qascade** [comment]
Hey, @harshsingh32 assigning this issue to you. Please rename the new branch to `feat.enclave_automation`. Please go through all the spec docs and guidelines of this project and stick to the conventions of this project properly.

**Comment by prakrit55** [comment]
hello @qascade, I wd like to work on it.

**Comment by qascade** [comment]
@prakrit55 Assigning this to you.

**Comment by prakrit55** [comment]
hey @qascade, do I need to create yaml files to automate the task ?

**Comment by qascade** [comment]
No, currently we are embedding temp_enclave.json and copying it on the go app just before compilation. Currently we are using CSV files to take input tables, but the file paths has to be mentioned before hand inside the enclave.json in files field. Otherwise service run fails. I want you to automate the csv filling inside the json to be automatic. You need to pick the csv filenames from source.yaml of Collaborators itself.

**Comment by prakrit55** [comment]
ohh, then where can I find the yamls ?

**Comment by qascade** [comment]
In samples folder
